akka-sse/src/test/scala/de/heikoseeberger/akkasse/ServerSentEventParserSpec.scala

```diff
@@ -27,20 +27,21 @@ class ServerSentEventParserSpec extends BaseSpec {
   "A ServerSentEventParser" should {
 
     "parse ServerSentEvents correctly" in {
-      val input = """|data: message 1 line 1
-                     |data:message 1 line 2
-                     |
-                     |data: message 2
-                     |:This is a comment and must be ignored
-                     |event: Only the last event should be considered
-                     |event: message 2 event
-                     |
-                     |data:
-                     |
-                     |event: message 4 event
-                     |
-                     |data: incomplete message
-                     |""".stripMargin
+      val input = List(
+        "data: message 1 line 1",
+        "data:message 1 line 2",
+        "",
+        "data: message 2",
+        ":This is a comment and must be ignored",
+        "event: Only the last event should be considered",
+        "event: message 2 event",
+        "",
+        "data:",
+        "",
+        "event: message 4 event",
+        "",
+        "data: incomplete message",
+        "").mkString("\n")
```

Comment:
Yes. I did that, and the problem still persisted. Then I noticed that every time I run SBT, a bunch of files get modified.
After disabling the header plugin by commenting:
https://github.com/hseeberger/akka-sse/blob/master/build.sbt#L8
https://github.com/hseeberger/akka-sse/blob/master/build.sbt#L13
https://github.com/hseeberger/akka-sse/blob/master/project/Build.scala#L5-6
https://github.com/hseeberger/akka-sse/blob/master/project/Build.scala#L65

And excluding the header plugin:
https://github.com/hseeberger/akka-sse/blob/master/project/Build.scala#L13

Everything works just fine.